### PR TITLE
5832 removing obsolete request action 'assumed'

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#5832

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Review by cameron-freshworks — APPROVED** [review]

File: client/src/components/new-request/submit-request/reserve-submit.vue [hunk 1]
```diff
@@ -96,10 +96,6 @@ export default class ReserveSubmitButton extends Vue {
 
     switch (setup) {
       case 'assumed':
-        if (this.$xproMapping['ASSUMED'].includes(entity_type_cd)) {
-          newReqModule.mutateRequestActionOriginal(request_action_cd)
-          newReqModule.mutateRequestAction('ASSUMED')
-        }
         newReqModule.mutateAssumedNameOriginal()
```
Comment (cameron-freshworks) [inline client/src/components/new-request/submit-request/reserve-submit.vue:99]: Normally the mutations take a parameter. Is this a method disguised as a mutation?

Comment (vysakh-menon-aot) [reply client/src/components/new-request/submit-request/reserve-submit.vue:99]: Yes

**Review by lekshmimallika-aot — COMMENTED** [review]
Is edit flow showing the expected labels for assumed flow?

**Comment by vysakh-menon-aot** [comment]
> Is edit flow showing the expected labels for assumed flow?

Yes

**Review by lekshmimallika-aot — APPROVED** [review]

**Comment by lekshmimallika-aot** [comment]
As part of another fix I noticed that this obsolete request action code mapping is used in multiple places. Example There is a watcher in names-capture.vue, in search.vue, new request module etc. Are these needed ?

**Comment by vysakh-menon-aot** [comment]
> As part of another fix I noticed that this obsolete request action code mapping is used in multiple places. Example There is a watcher in names-capture.vue, in search.vue, new request module etc. Are these needed ?

request action `assumed` is no longer valid. so any code related to that is redundant

**Pull request merged**